tests/lib/test_dump_load.py

```diff
@@ -0,0 +1,15 @@
+import yaml
+
+def test_dump(verbose=False):
+    assert yaml.dump(['foo'])
+test_dump.unittest = True
+
+def test_load_safeloader(verbose=False):
```

Comment:
Might want to add one that checks for `TypeError` if the second arg isn't specified?

Reply:
Done in 1b2f534